tests/utils.py

```diff
@@ -436,8 +436,8 @@ def __init__(self):
         super(DummyTracer, self).__init__()
         self._update_writer()
 
-    def _update_writer(self):
-        self.writer = DummyWriter()
+    def _update_writer(self, agent_url=None):
```

Comment:
For some reason (don't ask me why) this django test case started failing:

```
tracer = <tests.utils.DummyTracer object at 0x7fa6c69ae750>

    def test_configure_from_settings(tracer):
        pin = Pin.get_from(django)
    
        with override_config("django", dict()):
            assert "ddtrace.contrib.django" in django.conf.settings.INSTALLED_APPS
            assert hasattr(django.conf.settings, "DATADOG_TRACE")
    
            configure_from_settings(pin, config.django, django.conf.settings.DATADOG_TRACE)
    
            assert config.django.service_name == "django-test"
            assert config.django.cache_service_name == "cache-test"
            assert config.django.database_service_name_prefix == "db-test-"
            assert config.django.distributed_tracing_enabled is True
            assert config.django.instrument_databases is True
            assert config.django.instrument_caches is True
            assert config.django.analytics_enabled is True
            assert config.django.analytics_sample_rate is True
            # TODO: uncomment when figured out why setting this is not working
            # assert config.django.trace_query_string is True
    
            assert pin.tracer.enabled is True
            assert pin.tracer.tags["env"] == "env-test"
>           assert pin.tracer.writer.agent_url == "http://host-test:1234"
E           AssertionError: assert 'http://localhost:8126' == 'http://host-test:1234'
E             - http://localhost:8126
E             + http://host-test:1234
```

The failure is due to the test case using a dummy tracer which is reconfigured with the agent connection information. This failure was occurring because the dummy tracer has a dummy writer which prior to this change did not store the agent_url and so the connection information was never being updated on the writer, causing the test case to fail.